src/client/app/components/about.ts

```diff
@@ -29,9 +37,28 @@ import { environment, storageSize, Strings, storage } from '../helpers';
     `
 })
 
-export class About {
+export class About implements AfterViewInit {
     @Input() show: boolean;
     @Output() showChange = new EventEmitter<boolean>();
+
+    constructor(
+        private _effects: UIEffects,
+        private _changeDetector: ChangeDetectorRef
+    ) { }
+
+    ngAfterViewInit() {
+        if (environment.current.config.name === config['local'].name) {
+            this.configs.push({ name: 'local', value: 'Localhost' });
```

Comment:
Nit: Something about the capitalization of `Localhost` just looks weird.  What if, just for this one, we would write out its url (`config.editorUrl`).  I.e., `https://localhost:3000`.  But use the `editorUrl` variable, it's safer in case the ports ever change.  Or just `https://localhost`, that's fine too.